services/src/main/java/org/keycloak/protocol/oid4vc/issuance/OID4VCIssuerWellKnownProvider.java

```diff
@@ -58,14 +77,25 @@
  */
 public class OID4VCIssuerWellKnownProvider implements WellKnownProvider {
 
-    public static final String VC_KEY = "vc";
-
     private static final Logger LOGGER = Logger.getLogger(OID4VCIssuerWellKnownProvider.class);
 
-    protected final KeycloakSession keycloakSession;
+    // Realm attributes for signed metadata configuratio
+    public static final String SIGNED_METADATA_ENABLED_ATTR = "oid4vci.signed_metadata.enabled";
+    public static final String SIGNED_METADATA_LIFESPAN_ATTR = "oid4vci.signed_metadata.lifespan";
+    public static final String SIGNED_METADATA_ALG_ATTR = "oid4vci.signed_metadata.alg";
 
+    public static final String VC_KEY = "vc";
     public static final String ATTR_ENCRYPTION_REQUIRED = "oid4vci.encryption.required";
 
+    // Static list of supported asymmetric algorithms
+    private static final List<String> ASYMMETRIC_ALGORITHMS = Arrays.asList(
+            RS256, RS384, RS512,
+            ES256, ES384, ES512,
+            PS256, PS384, PS512
+    );
```

Comment:
Wouldn't it be better to get the available algorithms from the available keys in the realm instead of hardcoding them here?